scripts/prefs-coverage.py

```
@@ -0,0 +1,237 @@
+        return None
+    try:
+        res = subprocess.run(["git", "show", f"{base_sha}:{path}"],
+                              capture_output=True, timeout=120)
+    except subprocess.TimeoutExpired:
+        return None
+    if res.returncode != 0:
+        return None
+    return res.stdout.decode("utf-8", errors="replace")
+
+
+def main():
```

Ruff (0.15.20)

[warning] Too many return statements (11 > 6)

(PLR0911)